server/src/Models/userInsert.ts

```diff
@@ -0,0 +1,17 @@
+import { executeQuery } from "../utils/database";
+
+export const userSignUp = {
+    registerAsAdmin : async (user_email : string , hashedPassword : string, user_phone_number : string ,user_country : string , user_name : string) => {
+        const registerUser : string = "INSERT INTO `news_reader_app`.`users` (`user_email`, `user_password`, `user_phone_number`, `user_country` , `user_role` , `user_name`) VALUES (?,?,?,?,'admin',?)";
```

Comment:
try to find a better way than using ? for variables

Reply:
DONE ✅

Reply:
DONE ✅